docker/Dockerfile

```diff
@@ -1,13 +1,20 @@
-FROM ubuntu:xenial
+FROM debian:stretch
 
-RUN apt-get update \
+# Avoid ERROR: invoke-rc.d: policy-rc.d denied execution of start.
+RUN sed -i 's/^exit 101$/exit 0/' /usr/sbin/policy-rc.d \
+    && apt-get update \
     && apt-get install -y \
         curl \
+        gnupg2 \
         lsb-release \
+        procps \
+        systemd \
     && curl -sS https://dl.google.com/cloudagents/install-logging-agent.sh | bash \
     && apt-get autoremove \
     && sed -i "s/num_threads 8/num_threads 8\n  detect_json true\n  # Enable metadata agent lookups.\n  enable_metadata_agent true\n  metadata_agent_url \"http:\/\/local-metadata-agent.stackdriver.com:8000\"/" "/etc/google-fluentd/google-fluentd.conf" \
-    && rm -rf /var/lib/apt/lists/*
+    && rm -rf /var/lib/apt/lists/* \
+    && rm /tmp/* \
```

Comment:
`rm -rf`, just in case directories get created under `/tmp`?

Reply:
Done.